src/font.rs

```diff
@@ -580,27 +580,20 @@ impl Font {
     }
 
     /// Returns an iterator over all the glyph names _in the default layer_.
-    pub fn iter_names(&self) -> impl Iterator<Item = GlyphName> + '_ {
+    pub fn iter_names(&self) -> impl Iterator<Item = Name> + '_ {
+        //FIXME: why not &Name here?
```

Comment:
What do you mean?

Reply:
I mean why does the iterator return `Name` instead of `&Name`? I vaguely recall there was a reason at some point but it doesn't make sense to me now.

Reply:
I also don't. 

```text
error[E0271]: type mismatch resolving `<std::collections::btree_map::Keys<'_, name::Name, std::sync::Arc<glyph::Glyph>> as std::iter::Iterator>::Item == &&name::Name`      
   --> src\font.rs:583:33
    |
583 |     pub fn iter_names(&self) -> impl Iterator<Item = &Name> + '_ {
    |                                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ expected `&name::Name`, found struct `name::Name`
    |
    = note: expected reference `&&name::Name`
               found reference `&name::Name`
    = note: required because of the requirements on the impl of `std::iter::Iterator` 
for `std::iter::Cloned<std::collections::btree_map::Keys<'_, name::Name, std::sync::Arc<glyph::Glyph>>>`

For more information about this error, try `rustc --explain E0271`.
```

Reply:
that's just because there's a call to `clone()` on the iterator that clones the `Names`.